claim_validate_test.go

```diff
@@ -0,0 +1,705 @@
+package jwt
+
+import (
+	"crypto"
+	"errors"
+	"strconv"
+	"testing"
+	"time"
+)
+
+/*
+	Validator Creations
+*/
+
+// Test IssValidator function, it is not expected to return an error
+func TestIssValidator(t *testing.T) {
+	// Act
+	validator := IssuerValidator("testing")
+
+	// Assert
+	if validator == nil {
```

Comment:
Not really a useful check. Got to validate the happy flow plus each error scenario.